src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java

```diff
@@ -128,5 +190,16 @@ public boolean canBeManagedBy(Auth0UserProfile user) {
         return super.canBeManagedBy(user);
     }
 
+    public Itinerary lastItinerary() {
+        if (journeyState.responses.size() > 0) {
+            return journeyState.responses.get(0).plan.itineraries.get(0);
+        }
+        return itinerary;
+    }
```

Comment:
This seems to get the first itinerary. Please rename method and add some Javadoc.

Reply:
The idea is that it gets the most recently stored itinerary. Maybe latest is a better name.